testing/bdd/features/timing/computed_times.feature

```diff
@@ -0,0 +1,48 @@
+@timing @resolution @document
+Feature: Computed times computation
+
+  Examples:
+  | xml_file           | sequence_identifier | sequence_number |
+  | resolved_times.xml | testSequence1       | 1               |
+
+  # SPEC-CONFORMANCE: R16 R17
+  Scenario: Resolved times on a single document
+    Given an xml file <xml_file>
+    And it has sequenceIdentifier <sequence_identifier>
+    And it has timeBase <time_base>
+    And it has sequenceNumber <sequence_number>
+    And it has body begin time <body_begin>
+    And it has body end time <body_end>
+    And it has body duration <body_dur>
+    And it has div begin time <div_begin>
+    And it has div end time <div_end>
+    And it has p begin time <p_begin>
+    And it has p end time <p_end>
+    And it has span begin time <span_begin>
+    And it has span end time <span_end>
+    And the document is generated  # implicitly means it is valid.
+    And it has availability time <avail_time>
+    Then it has computed begin time <computed_begin>
+    And it has computed end time <computed_end>
+
+    Examples:
+    | time_base | body_begin  | body_end     | body_dur | div_begin   | div_end     | p_begin    | p_end      | span_begin  | span_end    | computed_begin | computed_end | avail_time  |
```

Comment:
Seems like this is a good start, but it would be useful to have more than one p, or span, say, as children of the same parent, to check that the `par` semantics are correctly being handled.